src/group_theory/quotient_group.lean

```diff
@@ -412,4 +413,42 @@ monoid_hom.to_mul_equiv
 
 end third_iso_thm
 
+
+section trivial
+
+/-These first few lemmas should maybe go somewhere else, where?-/
+
+lemma quot_by_top_is_subsingleton : subsingleton (quotient_group.quotient (⊤ : subgroup G)) :=
+trunc.subsingleton
+
+lemma left_rel_triv (H : subgroup G) (h : quotient_group.left_rel H = ⊤ ) :
+  ∀ x y : G , x⁻¹ * y ∈ H :=
+begin
+  let s := (quotient_group.left_rel H).r,
+  have h2 : ∀ x y : G, s x y ↔ x⁻¹ * y ∈ H, by {intros x y, refl},
+  intros x y,
+  have h3 := h2 x y,
+  rw ← h3,
+  simp_rw s,
+  rw h,
+  tauto,
+end
```

Comment:
This has a one-line proof,
```lean
lemma left_rel_triv {G} [group G] (H : subgroup G) (h : quotient_group.left_rel H = ⊤)
  (x y : G) : x⁻¹ * y ∈ H :=
(h.symm ▸ trivial : (quotient_group.left_rel H).rel x y)
```
although I think a better way to handle this would be to add the lemma
```lean
lemma setoid.eq_top_iff {α : Type*} {s : setoid α} : s = (⊤ : setoid α) ↔ ∀ x y : α, s.rel x y :=
by simp [eq_top_iff, setoid.le_def, setoid.top_def, top_apply]
```

Reply:
I've added this lemma and removed `left_rel_triv`.